lutron-status.js

```diff
@@ -17,20 +18,20 @@ module.exports = function (RED) {
           /*
           for dimmer action is always 1
           for pico action.param
-            FullOn => a=2, p=4
-            up     => a=5, p=4
-            down   => a=6  p=4
-            off    => a=4, p=4
+            FullOn => a=2, p=3
+            up     => a=5, p=3
+            down   => a=6  p=3
+            off    => a=4, p=3
```

Comment:
Pre-existing issue, but this is missing Button 2 (`a=3`), and it's also missing the buttons for the PJ2-4B model (`8`/`9`/`10`/`11`), which is rather a problem as that's the model I have 😅